_posts/2023-09-26-Hacktoberfest-2023.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "Hacktoberfest 2023"
+date:   2023-09-26
+tags:   Hacktoberfest
+author: rchakrab
+description: Article about Hacktoberfest participation and small descriptions about the participating projects and how to contribute in it.
+---
+
+= Hacktoberfest 2023
+
+image::hacktoberfest/hacktoberfest2023.png[Content Browser]
+
+https://hacktoberfest.com/[Hacktoberfest] is back, and it's time to channel your inner Open Source superhero! This annual event unites developers of all levels, from seasoned pros to enthusiastic beginners, in a shared mission to contribute to Open Source projects and celebrate the spirit of collaboration in software development. Join us for this month-long celebration of contributions as we work together, one pull request at a time.
+
+== What is Hacktoberfest?
+
+Hacktoberfest is a month-long event in October that encourages Open Source contributions and developers worldwide to give back to the global tech community. It's a fantastic opportunity for the contributors to showcase their skills and get to know about other Open Source communities.
+
+To participate you simply need to:
+
+
+* *Register:* https://hacktoberfest.com/auth/[Sign up] on the official Hacktoberfest website to get started.
+* *Discover Projects:* Explore the Hacktoberfest website to find a project that sparks your interest. You can also check out the curated list of projects we've prepared for you.
+* *Follow the Project guidelines:* Read the respective Contributing Guide, Code of Conduct, and follow best practices like forking the repository, creating a new branch, and making a meaningful contribution.
+* *Contribute:* Select an issue that matches your interest and start working on it. You'll have the option to receive a Hacktoberfest t-shirt or have a tree planted in your name as a token of appreciation.
```

Comment:
This year they are not gifting any t-shirts :( 
https://hacktoberfest.com/participation/

Reply:
Ah, nice catch. I'll update it.